src/main/kotlin/com/exactpro/th2/lwdataprovider/producers/GrpcMessageProducer.kt

```diff
@@ -37,6 +37,7 @@ class GrpcMessageProducer {
             return MessageGroupResponse.newBuilder().apply {
                 messageId = convertMessageId(storedMessage.id)
                 timestamp = convertTimestamp(storedMessage.timestamp)
+                putAllMessageProperties(storedMessage.metadata?.toMap() ?: emptyMap())
```

Comment:
Let's put properties only if we have them
```suggestion
                storedMessage.metadata?.toMap()?.let(this@apply::putAllMessageProperties)
```